cmd/firewall_delete.go

```diff
@@ -8,7 +8,8 @@ import (
 )
 
 func init() {
-	firewallDeleteCmd.Flags().BoolP("force", "f", false, "Attempt to remove security group without prompting for confirmation")
+	firewallDeleteCmd.Flags().BoolP("force", "f", false, "Remove security group without prompting for confirmation and delete all rules inside")
```

Comment:
IMHO it's not necessary to specify `and delete all rules inside` because rules can't live without their SG – and the current version doesn't specify it either...

Reply:
Now the difference with `force`  is the cli delete rules before deleting `sg` to prevent the error

Reply:
so i think change the way of the sentence to say delete rule and after sg

Reply:
See my other comment, but I currently don't have any error deleting a SG with rules in it without specifying `--force` (I just have to confirm "y" when prompted to do so).

Reply:
sorry but me I have the issu see my other comment thx